atlas/fixtures/basic/.atlas/atlas.toml

```diff
@@ -0,0 +1,15 @@
+# should atlas lookup proceed up the directory structure?
+#   default: false
+child = false
```

Comment:
child seems to arbitrary in this context. Can we make it something like `recurse_ancestor_atlases = false`